templates/_helpers.tpl

```diff
@@ -342,8 +342,8 @@ Merge packs and virtualenvs from st2 with those from st2packs images
     - 'sh'
     - '-ec'
     - |
-      /bin/cp -aR /opt/stackstorm/packs/. /opt/stackstorm/packs-shared &&
-      /bin/cp -aR /opt/stackstorm/virtualenvs/. /opt/stackstorm/virtualenvs-shared
+      /bin/cp -dR /opt/stackstorm/packs/. /opt/stackstorm/packs-shared &&
+      /bin/cp -dR /opt/stackstorm/virtualenvs/. /opt/stackstorm/virtualenvs-shared
```

Comment:
`-d` only preserves symlinks as-is. `-a` also copied the mode, including execute bits.

We need to make sure to preserve the execute bit on files. Would we have to add `--preserve=mode` to copy that?

Also, how will this interact with #245 (Switch from `cp` to `rsync` with fallback to `cp`)?

Reply:
Would you please check out #415? I'd like to come up with the minimal set of flags for `cp` and `rsync`.

Reply:
@Stealthii made these changes (which remains me I should credit him as an author in the commits..), I believe this was to fix issues with running as a non-privileged user. @Stealthii can you comment here / have a look at the above PRs :D

Reply:
@cognifloyd I've commented on #415 with a proposed change that should preserve all relevant attributes besides ownership. https://github.com/StackStorm/stackstorm-k8s/pull/415#issuecomment-2100894943

Once verified and if it becomes part of that PR, we can drop the change here and have this PR focus solely on the enablement of non-root runtimes.